build.gradle

```diff
@@ -17,7 +17,7 @@ allprojects {
 
 	repositories {
 		mavenCentral()
-		if (!version.endsWith('RELEASE')) {
+		if (version =~ /((-M|-RC)[0-9]+|-SNAPSHOT)$/) {
```

Comment:
Thanks, Roy. I think we could simplify this to just look for a `-` in the version:

```java
if (version.contains('-')) {
```

WDYT?

Reply:
Indeed, that works as well. I suppose there's no real benefit to being more specific with the regex in my commit.